gcp/website/blog/content/posts/2024-in-review/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "The Year in Review"
+date: 2024-12-13T04:00:00Z
+draft: false
+author: The OSV Team
+---
+2024 has been an *even more* eventful year for OSV.
```

Comment:
we should also add https://security.googleblog.com/2024/12/google-cloud-expands-vulnerability.html somewhere now that it's published.

Reply:
I've added a new "Integrations" section under "Community" and linked to this blog post. Ideally we'd have some other noteworthy call outs in here...